examples/reqwest-report.rs

```diff
@@ -0,0 +1,105 @@
+use threescalers::api_call::*;
+use threescalers::application::*;
+use threescalers::credentials::*;
+use threescalers::extensions::Extensions;
+use threescalers::http::request::FromRequest;
+use threescalers::http::Request;
+use threescalers::service::*;
+use threescalers::transaction::Transaction;
+use threescalers::usage::Usage;
+
+use reqwest::Client;
+use reqwest::{RequestBuilder, Response};
+
+fn main() -> Result<(), threescalers::errors::Error> {
+    let creds = Credentials::ServiceToken(ServiceToken::from("12[3]token"));
+    let svc = Service::new("svc123", creds);
+    let uks = [
+        "userkey_1",
+        "userkey_2",
+        "userkey_3",
+        "userkey 4",
+        "userkey 5",
+    ];
+    let apps = uks
+        .into_iter()
+        .map(|uk| Application::from(UserKey::from(*uk)))
+        .collect::<Vec<_>>();
+
+    println!("Apps: {:#?}", apps);
```

Comment:
The output of this seems to have an extra `UserKey`.

```
Apps: [                                                                                                                                                                         
    UserKey(                                                                                                                                                                    
        UserKey(                                                                                                                                                                
            "userkey_1",                                                                                                                                                        
        ),                                                                                                                                                                      
    ),                                                                                                                                                                          
    UserKey(                                                                                                                                                                    
        UserKey(                                                                                                                                                                
            "userkey_2",                                                                                                                                                        
        ),                                                                                                                                                                      
    ), 
```

Reply:
this is correct, it is a debug view of `Application::UserKey(UserKey)`

Reply:
(perhaps we might want to look into whether we'd want to change/simplify it in a different PR)

Reply:
Yes, we should :+1: